src/vl-map-action.js

```diff
@@ -70,4 +71,10 @@ export class VlMapAction extends vlElement(HTMLElement) {
       this.setAttribute('active', this._mapElement.activeAction == this.action);
```

Comment:
mijn ide gaf ook al aan == te veranderen naar ===. Weet niet of het relevant is.

Reply:
Kan zeker geen kwaad om `===` toe te passen, dus zal het wijzigen.